apis/apps/v1alpha1/cluster_types.go

```diff
@@ -833,6 +832,12 @@ type ClusterComponentService struct {
 	//
 	// +optional
 	Annotations map[string]string `json:"annotations,omitempty"`
+
```

Comment:
there are a few issues about clusterComponentService:

1. The semantics of ClusterComponentService.Name has changed. In the new implementation, it should correspond to the name of the services defined in the ComponentDefinition. This should be noted in the API documentation.
2.Should users be allowed to override the roleSelector in the ClusterComponentService?
3.With the reuse of ClusterComponentService, is the ComponentSelector in ClusterService still necessary?

Reply:
Considering that the current API will be deprecated and removed in the latest version, the Service field is directly reused here without introducing a new one. 
In fact, they have different semantics under the new and legacy APIs. In the new API, it can only override some of the predefined behaviors or attributes of the service defined in the ComponentDefinition, and can no longer be used to define new services as before. Therefore, there is no conflict or substitution between it and the cluster service.